modules/hostdb.go

```diff
@@ -40,8 +40,9 @@ type HostSettings struct {
 // coins frozen in the host's announcement transaction.
 type HostEntry struct {
 	HostSettings
-	IPAddress NetAddress
-	Weight    types.Currency
+	IPAddress   NetAddress
+	Weight      types.Currency
+	Reliability types.Currency
```

Comment:
Gives us more flexibility and access to unbounded unsigned integers. Reliability right now could just be a uint8 but I'm not sure how that's going to change in the future.

Weight on the other hand needs a lot of bits. It could probably just be a Big but things would break if there were negative numbers, I'd prefer to have the unsigned safety of the currency type.

I do need to take a second look at the equations, maybe there's a better way to do things. Maybe I should be using big.Rat everywhere.